### PR TITLE
Polish up for release

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: adafruit_ble_eddystone/url.py [hunk 1]
```diff
@@ -34,61 +34,56 @@
...
-    b'.com',
-    b'.org',
-    b'.edu'
-    b'.net',
-    b'.info',
-    b'.biz',
-    b'.gov',
+    b".com",
+    b".org",
+    b".edu" b".net",
```
Comment (dhalbert) [inline adafruit_ble_eddystone/url.py]: Missing comma here, I think.

Comment (dhalbert) [reply adafruit_ble_eddystone/url.py]: it got reformatted to one line.

**Review by dhalbert — CHANGES_REQUESTED** [review]
I'm not so keen on some of black's changes, like double quotes for single quotes or uppercasing hex constants, but oh well.

**Comment by tannewt** [comment]
This is blocked on https://github.com/adafruit/Adafruit_Blinka_bleio/pull/1 which will introduce a richer `_bleio`

**Review by dhalbert — APPROVED** [review]
Looks good!

**Pull request merged**